### DISCUSSION TITLE
fail to quit the driver after taking screenshot on iOS

### DISCUSSION THREAD

**Discussion by earlif** [appium/appium#16249] [General]
I am using java to test APP on iOS. And after calling 
File screenFile = driver.getScreenshotAs(OutputType.FILE);
to take screenshot, it fails to quit the driver
driver.quit();
And it is due to the: Cannot read properties of null (reading 'webDriverAgentUrl').

**Comment by jlipps** [comment]
This seems like a bug report, not a question. Please fill out an issue template with all requested logs and information.
